openapi_python_client/config.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from typing import Dict, Optional
+
+import yaml
+from pydantic import BaseModel
+
+
+class ClassOverride(BaseModel):
+    class_name: str
+    module_name: str
+
+
+class Config(BaseModel):
+    class_overrides: Optional[Dict[str, ClassOverride]]
+    project_name_override: Optional[str]
+    package_name_override: Optional[str]
+
+    def load_config(self) -> None:
+        """ Loads config from provided Path """
+
+        if self.class_overrides is not None:
+            from .parser import reference
+
+            for class_name, class_data in self.class_overrides.items():
+                reference.class_overrides[class_name] = reference.Reference(**dict(class_data))
+
+        from openapi_python_client import _Project
+
+        _Project.project_name_override = self.project_name_override
+        _Project.package_name_override = self.package_name_override
```

Comment:
Accessing an internal (begins with underscore) class is considered bad form. Also, class variables are still effectively globals, just scoped to a class namespace instead of a module namespace. The problem with them is that they aren't stored within the context of a function, which means they have lasting effects outside of their function. It's probably not the biggest issue given this is a single threaded CLI tool, it could really only bite us during unit tests if things are properly reset.

I think just rename `_Project` to `Project` for now and I'll do a big sweep to remove all globals in a future version.